### PR TITLE
Add CodesUI

### PR DESCRIPTION
**Pull request by dtip**
WIP

### CODE REVIEW

**Comment by dtip** [comment]
Looks like we've got our green ticks.

@conda-forge/help-c-cpp could you please review?

File: recipes/codes-ui/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+about:
+  home: https://confluence.ecmwf.int/display/METV/CodesUI
+  license: Apache 2.0
+  license_family: Apache
+  summary: ECMWF CodesUI Copyright 2007-2019 ECMWF.
+
+extra:
+  recipe-maintainers:
+    - dtip
+    - StephanSiemen
```
Comment (chrisburr) [inline recipes/codes-ui/meta.yaml]: @StephanSiemen Are you happy to be listed as a maintainer?

File: recipes/codes-ui/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+test:
+  commands:
+    - conda inspect linkages -p $PREFIX codes-ui
+    - conda inspect objects -p $PREFIX codes-ui  # [osx]
+
+about:
+  home: https://confluence.ecmwf.int/display/METV/CodesUI
+  license: Apache 2.0
+  license_family: Apache
```
Comment (chrisburr) [inline recipes/codes-ui/meta.yaml]: I'm not certain if the Apache 2.0 license requires it, but you should probably include it in the package using:
```suggestion
  license_family: Apache
  license_file: metview/src/codes_ui/LICENSE
```

Comment (dtip) [reply recipes/codes-ui/meta.yaml]: Didn't know you could do this. Thanks :)

File: recipes/codes-ui/meta.yaml (hunk 1 above)
Comment (StephanSiemen) [reply recipes/codes-ui/meta.yaml]: @chrisburr , yes I am happy to be a maintainer for they package

Comment (chrisburr) [reply recipes/codes-ui/meta.yaml]: Great, LGTM 👍

**Pull request merged**